code/game/objects/structures/crates_lockers/closets/job_closets.dm

```diff
@@ -67,18 +67,34 @@
 /obj/structure/closet/jcloset/New()
 	..()
 	new /obj/item/clothing/under/rank/janitor(src)
+	new /obj/item/clothing/under/rank/janitor(src)
+	new /obj/item/device/radio/headset/headset_service(src)
 	new /obj/item/device/radio/headset/headset_service(src)
 	new /obj/item/weapon/cartridge/janitor(src)
+	new /obj/item/weapon/cartridge/janitor(src)
+	new /obj/item/clothing/gloves/black(src)
 	new /obj/item/clothing/gloves/black(src)
 	new /obj/item/clothing/head/soft/purple(src)
+	new /obj/item/clothing/head/soft/purple(src)
+	new /obj/item/clothing/head/beret/purple(src)
 	new /obj/item/clothing/head/beret/purple(src)
 	new /obj/item/device/flashlight(src)
+	new /obj/item/device/flashlight(src)
+	new /obj/item/weapon/caution(src)
+	new /obj/item/weapon/caution(src)
+	new /obj/item/weapon/caution(src)
+	new /obj/item/weapon/caution(src)
 	new /obj/item/weapon/caution(src)
 	new /obj/item/weapon/caution(src)
 	new /obj/item/weapon/caution(src)
 	new /obj/item/weapon/caution(src)
 	new /obj/item/device/lightreplacer(src)
+	new /obj/item/device/lightreplacer(src)
+	new /obj/item/weapon/storage/bag/trash(src)
 	new /obj/item/weapon/storage/bag/trash(src)
+	new /obj/item/weapon/storage/belt/janitor(src)
+	new /obj/item/weapon/storage/belt/janitor(src)
+	new /obj/item/clothing/shoes/galoshes(src)

```

Comment:
The room is a bit awkwardly shaped but it seems like a better idea to just map in one more of each of the changed closets. Just my opinion tho.
Lockers do magically adjust their size according to their contents (on spawn) if there's too much but I've always felt that was a bit of a hack.